docs/core/linux-prerequisites.md

```diff
@@ -103,537 +105,279 @@ On Linux, there are two installer package choices:
 
-   ```bash
-   sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
-   ```
+**Fedora 23**
 
-5. Add dotnet to your PATH.
+* .NET Core Runtime 1.0.9 [install link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-1.0.9-linux-fedora-23-x64-binaries)
+* .NET Core SDK 1.0.4 [install link](https://www.microsoft.com/net/download/thank-you/dotnet-sdk-1.0.4-linux-fedora-23-x64-binaries)
+* .NET Core SDK 1.0.1 [install link](https://www.microsoft.com/net/download/thank-you/dotnet-sdk-1.0.1-linux-fedora-23-x64-binaries)
 
-   ```bash
-   sudo ln -s /opt/dotnet/dotnet /usr/local/bin
-   ```
-   
 ---
 
-6. Run the `dotnet --version` command to prove the installation succeeded.
+## Install .NET Core for supported CentOS and Oracle Linux distributions/versions (64 bit)
 
-   ```bash
-   dotnet --version
-   ```
-
-## Install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit)
-
-To install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit):
-
-1. Remove any **previous preview** versions of .NET Core from your system.
+To install .NET Core for supported CentOS and Oracle Linux distributions/versions (64 bit):
 
 > [!NOTE]
 > A user-controlled directory is required for Linux system installs from tar.gz.
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-2. Register the Microsoft signature key.
-
-   ```bash
-   sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-   ```
-
-3. Add the Microsoft Product feed.
+1. Remove any **previous preview** versions of .NET Core from your system.
 
-   ```bash
-   sudo sh -c 'echo -e "[packages-microsoft-com-prod]\nname=packages-microsoft-com-prod \nbaseurl=https://packages.microsoft.com/yumrepos/microsoft-rhel7.3-prod\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/yum.repos.d/dotnetdev.repo'
-   ```
+2. Install .NET Core 2.x on supported CentOS and Oracle Linux distributions/versions (64 bit):
 
-4. Install the .NET Core SDK.
+**.NET Core 2.0**
 
-   ```bash
-   sudo yum update
-   sudo yum install libunwind libicu
-   sudo yum install dotnet-sdk-2.0.0
-   ```
+* .NET Core Runtime 2.0.6 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/runtime-2.0.6)
+* .NET Core Runtime 2.0.5 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/runtime-2.0.5)
+* .NET Core SDK 2.1.103 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/sdk-2.1.103)
+* .NET Core SDK 2.0.3 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/sdk-2.0.3)
+ 
+**.NET Core 2.1**
 
-5. Add dotnet to your PATH
+>[!IMPORTANT]
+> To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=banner+cta&utm_content=vs2017).
```

Comment:
in this case, the non-campaign link was better: https://www.visualstudio.com/vs/preview/